src/api/faq/faq.controller.js

```diff
@@ -8,20 +8,14 @@ async function getFAQ(req, res, next) {
       try {
         response = [await FAQ.findById(req.params.id).exec()];
         if (response[0] === null) throw new ReferenceError('The requested ID does not exist.');
-      } catch ({
-        message,
-      }) {
+      } catch ({ message }) {
         throw new ReferenceError(message);
       }
     } else response = await FAQ.find({}).exec();
 
     const finalResponse = response
       .map(({
-        id,
-        question,
-        answer,
-        createdAt,
-        updatedAt,
+        id, question, answer, createdAt, updatedAt,
```

Comment:
I prefer the newlines for imports and destructuring. I will try to find a linter rule or prettier rule to enforce this.

Reply:
OK that would be good. They were just getting messy and inconsistent.

Reply:
I would be nice to have specific rules on how the code should be formatted.